CONTRIBUTING.md

```diff
@@ -35,10 +35,13 @@ Follow [this link](https://circleci.com/account/api) for generate your CircleCI
 
 ### First Install App
 ```bash
-$ make init
+$ make start
```

Comment:
is it not useful `make init` to start the app?
if not, we can change the name of this section because it's not the "first install" in that case.

What do you think?

Reply:
Right! With the new docker-compose when it up, it install and build all dependecies!

Reply:
Fixed!